sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

```diff
@@ -122,7 +123,15 @@ object JdbcUtils extends Logging {
             }
             i = i + 1
           }
-          stmt.executeUpdate()
+          stmt.addBatch()
+          rowCount += 1
+          if (rowCount % 10000 == 0) {
```

Comment:
I thought it might not worth adding a configuration for it before. But seems it is hard to decide a proper batch size now. So I add a configuration as you suggested. Thanks.